File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.TrainingService.Models/save-training-event-roster_param.ts


import { ITrainingEventRoster_Item } from './itraining-event-roster_item';
import { ISaveTrainingEventRoster_Param } from './isave-training-event-roster_param';

export class SaveTrainingEventRoster_Param implements ISaveTrainingEventRoster_Param {
  
	public TrainingEventID: number = 0;
	public TrainingEventGroupID?: number;
	public ParticipantType: string = "";
	public Participants?: ITrainingEventRoster_Item[];
	public StudentExcelStream?: any;
  
	public ErrorMessages: string[] = null;
}

File: GTTS/INL.TrainingService.Models/SaveTrainingEventRoster_Param.cs
using System.IO;
using System.Collections.Generic;
using INL.Services.Models;

namespace INL.TrainingService.Models
{
    public class SaveTrainingEventRoster_Param : BaseParam, ISaveTrainingEventRoster_Param
    {
        public long TrainingEventID { get; set; }
        public long? TrainingEventGroupID { get; set; }
        public string ParticipantType { get; set; }
        public List<ITrainingEventRoster_Item> Participants { get; set; }
        public Stream StudentExcelStream { get; set; }
        public int? ModifiedByAppUserID { get; set; }
    }
}
